src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/BuildPropertyPage/DefaultPlatformPropertyValueProvider.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties
+{
+    [ExportInterceptingPropertyValueProvider("PlatformTarget", ExportInterceptingPropertyValueProviderFile.ProjectFile)]
+    internal sealed class DefaultPlatformPropertyValueProvider : InterceptingPropertyValueProviderBase
+    {
+        private const string AnyCPUCsproj = "AnyCPU";
+        private const string AnyCPUPropertyPages = "Any CPU";
```

Comment:
Acronyms appearing in .NET identifiers and having more than two characters are in Pascal case (e.g. `AnyCpuCsproj` and `AnyCpuPropertyPages`).

Also `AnyCPU` is not specific to csproj.

I'd suggest using `AnyCpuPlatformName` and `AnyCpuDisplayName` for these constants.